extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -327,56 +333,32 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 				}
 				return errReason;
 			}
-
-			function checkItemsForErrors($items, errTitle, errReason, reasonMessage) {
-				$items.each(function () {
-					if (this.value === errTitle) {
-						var $itemWithError;
-
-						switch (errReason) {
-							case 'missingImage':
-								$itemWithError = $(this).parent().find('.image');
-								break;
-							case 'emptyLabel':
-							case 'tooLongLabel':
-								$itemWithError = $(this).next();
-								break;
-							default:
-								$itemWithError = $(this);
-						}
-						if ($itemWithError) {
-							$itemWithError.addError(reasonMessage);
-							return false;
-						}
-					}
-					return true;
-				});
+			function addErrorToItem($selector, errReason, reasonMessage) {
+				switch (errReason) {
+					case 'imageMissing':
+						$selector.find('.image').addError(reasonMessage);
+						break;
+					case 'emptyLabel':
+						// intended fall
+					case 'duplicatedLabel':
+						// intended fall
+					case 'tooLongLabel':
+						$selector.find('.name').addError(reasonMessage);
+						break;
+					default:
+						$selector.find('.item-input').addError(reasonMessage);
+				}
 			}
-			function checkSectionsForErrors($sections, errTitle, errReason, reasonMessage) {
-				$sections.each(function () {
-					if (this.value === errTitle) {
-						var $itemWithError;
-
-						switch (errReason) {
-							case 'missingImage':
-								$itemWithError = $(this).parent().find('.image');
-								break;
-							case 'duplicatedLabel':
-							// intended fall
-							case 'tooLongLabel':
-								$itemWithError = $(this);
-								break;
-						}
-						if ($itemWithError) {
-							$itemWithError.addError(reasonMessage);
-							return false;
-						}
-					}
-					return true;
-				});
+			function addErrorToSection($selector, errReason, reasonMessage) {
+				switch (errReason) {
```

Comment:
What would be benefit from that? I wanted it to be as similar to `addErrorToItem` as possible - do you think of any reason why I should change it so much?